soda/src/main/java/it/okkam/opendata/models/soda/SodaResource.java

```diff
@@ -4,24 +4,28 @@
 
   String name;
   String id;
-  String[] parent_fxf;
   String description;
   String attribution;
-  String attribution_link;
-  String contact_email;
   String type;
   String updatedAt;
   String createdAt;
   String metadata_updated_at;
   String data_updated_at;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "data_updated_at" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00116)